src/sentry/api/bases/organization.py

```diff
@@ -430,7 +433,11 @@ def get_filter_params(
         try:
             if isinstance(project_ids, list):
                 project_ids = set(project_ids)
-            projects = self.get_projects(request, organization, project_ids=project_ids)
+            if isinstance(project_slugs, list):
```

Comment:
if you pass both you will always use the slug. do we want to enforce that you only pass one?

Reply:
oh i see

Reply:
Doest that mean if I pass in an empty list/set for project_ids, but populated project_slugs, slugs will be ignored and I'll get an empty list of projects? To ensure slugs get used I need to make sure project_ids is `None`?

Reply:
Yup correct

both project_ids and project_slugs are defaulted to `None` so you would just avoid passing those in as args

Now for the fun part
Apparently we decided some time ago that empty list `[]` means I want to fetch all the project's that i am a member of.
project id of `-1` means all projects i have access to (or project slug of `$all`)

So - if you were to pass an empty list/set for project_ids, that _would_ be an explicit query 😬